WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,3 +1,6 @@
-- The Quick Start card and Next Steps tours match the new designs for existing sites.
-- Insert images from other sites using the “Insert from URL” option in the app.
-- Get more context through your screen reader with block editor accessibility improvements.
+* [*] Site creation: enhances the design selection screen with recommended designs [https://github.com/wordpress-mobile/WordPress-Android/pull/16468]
+* [*] DeepLinks: fix auto-verification of App Links in Android 12 [https://github.com/wordpress-mobile/WordPress-Android/pull/16696]
+* [*] Quick Start: Fix Quick Start focus points in RTL mode [https://github.com/wordpress-mobile/WordPress-Android/pull/16720]
+* [*] [internal] Block Editor: Bump react-native-gesture-handler to version 2.3.2. [https://github.com/wordpress-mobile/WordPress-Android/pull/16645]
+* [***] Jetpack App: Introducing blogging prompts. Build a writing habit and support creativity with a periodic prompt for inspiration. [https://github.com/wordpress-mobile/WordPress-Android/pull/16729]
```

Comment:
When I read this in your PR description I was 🤔 because I noticed from the announcement post that they were in fact different

> [x] WordPress/jetpack_metadata/release_notes.txt audited to remove any item not applicable for Jetpack.
>    - Nothing WP or JP specific this time around at least to my understanding, so kept them the same

But I see that you properly only kept the "Jetpack App:" item only on the JP release notes and properly removed it from the WP release notes, so you did end up making them different this time just as was expected 😅 👍 

I was wondering:
 - Where did you get the checklist for this PR's description: did you copy it from one of my previous PR, or got it from this internal post paaHJt-2WJ-p2#comment-5741 or somewhere else?
 - In any case, is the wording especially of this checklist item clear? I realized that the item mention "remove items not applicable to JP from the JP notes" but in practice it also applies to the other way around, removing items applicable only to the JP app from the WP notes, just like you did here. So maybe we should update the wording of that checklist?